fetch.bs

```diff
@@ -6667,11 +6676,16 @@ constructor steps are:
 
  <li><p>If <var>init</var> is given, then <a for=Headers>fill</a> <a>this</a> with <var>init</var>.
 </ol>
+</div>
 
+<div algorithm>
 <p>The <dfn export for=Headers method><code>append(<var>name</var>, <var>value</var>)</code></dfn>
 method steps are to <a for=Headers>append</a> (<var>name</var>, <var>value</var>) to <a>this</a>.
+</div>
 
-<p>The <dfn export for=Headers method><code>delete(<var>name</var>)</code></dfn> method steps are:
+<div algorithm>
+<p>The <dfn export for=Headers method><code>delete(<var>name</var>)</code></dfn> method
+from a {{Headers}} object (<var>headers</var>), run these steps:
```

Comment:
You found a bug, nice! The fix is replacing _headers_ below with `<a>this</a>`. This can stay as-is. I suppose the same applies to `set()` below.

Reply:
Ah! Good point. Originally I thought this would be like [remove privileged no-CORS request-headers](https://github.com/whatwg/fetch/pull/1596/files#diff-b4827ee42aceaf9dc3e7216f72262d81c32217e9996fb70a657f259e3ac92604L6647), but it makes sense that we'd use `this` for these methods.